monarchmoney/monarchmoney.py

```diff
@@ -2352,14 +2392,54 @@ async def set_budget_amount(
             graphql_query=query,
         )
 
+    async def upload_and_parse_balance_history(
+        self,
+        account_id: str,
+        csv_content,
+        timeout: int = 300,
+        delay: int = 10,
```

Comment:
define these as constants, so it can be consistent across the API

Reply:
done!